includes/gateway/class-omise-payment.php

```diff
@@ -267,6 +277,104 @@ public function sync_payment( $order ) {
 		}
 	}
 
+	/**
+	 * Set an order transaction id
+	 *
+	 * @param string $transaction_id  Omise charge id.
+	 */
+	protected function set_order_transaction_id( $transaction_id ) {
+		/** backward compatible with WooCommerce v2.x series **/
+		if ( version_compare( WC()->version, '3.0.0', '>=' ) ) {
+			$this->order()->set_transaction_id( $transaction_id );
+			$this->order()->save();
+		} else {
+			update_post_meta( $this->order()->id, '_transaction_id', $transaction_id );
+		}
+	}
+
+	/**
+	 * @param int|mixed $order_id
+	 */
+	protected function payment_failed_no_order( $order_id ) {
+		wc_add_notice(
+			sprintf(
+				wp_kses(
+					__( 'We cannot process your payment.<br/>Note that nothing wrong by you, this might be from our store issue.<br/><br/>Please feel free to try submit your order again or report our support team that you have found this problem (Your temporary order id is \'%s\')', 'omise' ),
```

Comment:
We have been unable to process your payment.<br/>Please note that you've done nothing wrong - this is likely an issue with our store.<br/><br/>Feel free to try submitting your order again, or report this problem to our support team (Your temporary order id is \'%s\')